_2019_pages/venue.html

```diff
@@ -12,15 +12,250 @@
+              Due to extensive construction works at Damaschkeplatz and around
+              the central railway station, take a 10 minutes walk from the main
+              exit of the railway station to central tram station <strong>Alter
+              Markt</strong>.
+          </li>
+          <li>
+              From tram station <strong>Alter Markt</strong>, take
+              <a href="https://www.mvbnet.de/verkehr/fahrplan/fahrplaene-strassenbahn/linie-5-fahrplanwechsel-2018-1/">
+                  <strong>tram number 5</strong>
+              </a>
+              in direction <strong>Messegelände/Elbauenpark</strong> and get off
+              at station <strong>Askanischer Platz</strong> (a 10 minutes ride).
+          </li>
+          <li>
+              From there, walk over the crossroad (Walther-Rathenau-Straße)
+              towards Sandtorstraße where you will find the Max Planck
+              Institute on the right side.
+          </li>
+          <li>
+              Current timetables are available at
+              <a href="https://www.mvbnet.de/">Magdeburger Verkehrsbetriebe</a>.
+          </li>
+      </ul>
+
+      <h3>By car</h3>
+      <ul>
+          <li>
+              Leave the motorway (Autobahn) A2 at exit
+              <strong>Magdeburg-Zentrum</strong> or A14 at exit
+              <strong>Magdeburg-Sudenburg</strong>.
+          </li>
+          <li>
+              Turn onto B189/B71 (Magdeburger Ring) and leave at exit
+              <strong>Zentrum/Universität</strong>, direction
+              <strong>B1 (Burg/Dessau)</strong>.
+          </li>
+          <li>
+              Follow the street signs towards B1 and go straight through the
+              tunnel Universitätsplatz and continue on B1.
+          </li>
+          <li>
+              Turn left towards Rothensee/Hafen into Sandtorstraße at the
+              traffic lights before crossing the bridges.
+          </li>
+          <li>The Max Planck Institute is directly on the right side.</li>
+          <li>Parking space is available at the institute.</li>
+      </ul>
 
-      <p>Info</p>
+      <h3>Useful links</h3>
+      <ul>
+          <li>
+              <a href="https://www.nasa.de">
+                  Connections in Germany (bus, train, local traffic)
+              </a>
+          </li>
+          <li>
+              <a href="https://www.mvbnet.de/verkehr/liniennetzplaene/">
+                  Surveys of tram and bus lines in Magdeburg
+              </a>
+          </li>
+          <li>
+              <a href="https://www.magdeburg.de/">
+                  Information about Magdeburg
+              </a>
+          </li>
+      </ul>
 
-      <p>Address</p>
+      <h2>Accomodation</h2>
+      <p>
+          If you are interested for a limited, free accommodation offer, please
+          indicate this in your registration email.
+      </p>
 
-      <p>Location</p>
+      <p>
+          Here are some hotel/hostel options in Magdeburg:
```

Comment:
How limited is space in the guest house? If it is reasonable to assume that all participants will be able to stay there, I would remove the Hotel list and do more advertising for that.

Reply:
We don't know exactly how many places will be available. This ("limited" and "free") would hopefully encourage people to register early.